jsonschema2pojo-ant/src/main/java/org/jsonschema2pojo/ant/Jsonschema2PojoTask.java

```diff
@@ -668,6 +670,18 @@ public void setTargetVersion(String targetVersion) {
     public void setIncludeDynamicAccessors(boolean includeDynamicAccessors) {
         this.includeDynamicAccessors = includeDynamicAccessors;
     }
+    
+    /**
+     * Sets the 'formatDateTime' property of this class
+     *
+     * @param formatDateTime
+     *            Whether the fields of type `date-type` have the `@JsonFormat` annotation 
```

Comment:
I think `date-type` should be `date-time`. Also, I think the backticks should be `<code></code>` (?).